inst/stan/epinowcast.stan

```diff
@@ -39,6 +35,11 @@ data {
   matrix[urds, nrd_effs + 1] rd_fixed; // design matrix for report dates
   int nrd_eff_sds; // number of standard deviations to use for pooling for rds
   matrix[nrd_effs, nrd_eff_sds + 1] rd_random; // Pooling pmf design matrix 
+  // Observations
+  int obs[s, dmax]; // obs for each primary date (row) and report date (column)
+  int flat_obs[n]; // obs stored as a flat vector
+  int obs_miss[g, t]; // obs with missing primary date (group first)
+  int latest_obs[t, g]; // latest obs for each snapshot group
```

Comment:
A currently unresolved issue with the latest obs is that the cases with missing reference dates are not included, slightly biasing the seeding. We have to decide on a sufficiently accurate, not to complicated way to include them. One way would be to do a rough back-projection (either assuming some backward reporting delay distribution or taking the empirical one). Should be discussed.